Example/Marker/AppDelegate.swift

```diff
@@ -11,11 +11,23 @@ import UIKit
 @UIApplicationMain
 class AppDelegate: UIResponder, UIApplicationDelegate {
 
+    // MARK: - Properties
+    
     var window: UIWindow?
+    
+    // MARK: - Private properties
+    
+    private var theme: AppTheme!
```

Comment:
if you're gonna save it, instantiate it here too? unless you want to add an IoC framework huehuehue